src/main/java/com/microsoft/sqlserver/jdbc/Geography.java

```diff
@@ -203,8 +216,8 @@ public boolean hasZ() {
      * @return double value that represents the X coordinate.
      */
     public Double getX() {
```

Comment:
Please rename getX() to getLatitude() and getY() to getLongitude() for Geography.
A Geographic Point should always reference them as Latitude and Longitude. There should be no API neither documentation for these co-ordinates as X or Y in context of Geography Point.